.git-hooks/pre-push

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+git diff HEAD --name-only --diff-filter=ACM | grep '\(resources\|metadata\).json$' | xargs npm run prepush --quiet
```

Comment:
Ah, I see your explanation above now

> executes `npm test` before push if either `metadata.json` or `dist/resources.json` have changes

That list should at least include `filter_lists/default.json` and `filter_lists/regional.json`, although I think I'd prefer to just run it on _any_ push just in case some of the file structure changes. The tests finish extremely quickly anyways :smile:

Reply:
Great! Running every time makes much more sense now. Just updated with this change :smile: 

I have restricted to those files due to the README instructions, perhaps we could add that the tests are executed before every push now?